libcloud/backup/drivers/dimensiondata.py

```diff
@@ -180,7 +180,8 @@ def create_target_from_container(self, container,
         return NotImplementedError(
             'create_target_from_container not supported for this driver')
 
-    def update_target(self, target, name=None, address=None, extra=None):
+    def update_target(self, target, name=None, address=None, extra=None,
+                      type=BackupTargetType.VIRTUAL):
```

Comment:
`type` argument is not part of the standard API so it should be prefixed with `ex_`, right?